src/components/BulkEditStartModal/BulkEditStartModal.js

```diff
@@ -77,11 +77,13 @@ const BulkEditStartModal = ({
       setIsFileUploaded(true);
 
       setConformationButton(false);
-    } catch {
-      showCallout({
-        message: errorMessage,
-        type: 'error',
-      });
+    } catch (error) {
```

Comment:
looks like error isn't used. Mb we can remove it

Reply:
I changed a bit of logic to use `error` instead previous `ref`